py/desiperf/instperfapp/pages/positioner.py

```diff
@@ -29,15 +29,16 @@ def __init__(self, datahandler):
         posfiles = glob.glob(os.getcwd()+'/instperfapp/data/per_fiber/*.csv')
         self.pos_list = [os.path.splitext(os.path.split(posf)[1])[0] for posf in posfiles] #np.linspace(0,4999,5000)
         self.x_options = ['OFFSET_0','OFFSET_FINAL']
```

Comment:
The .csv files do not have a 'OFFSET_FINAL' column

Reply:
Great catch! I believe they are OFFSET_2 or similar. I can make this quick change!

Reply:
Yes, it is 'OFFSET_2'